compiler/x/env/OMRCPU.cpp

```diff
@@ -144,5 +150,8 @@ OMR::X86::CPU::getX86OSSupportsSSE2(TR::Compilation *comp)
 bool
 OMR::X86::CPU::getX86SupportsTM(TR::Compilation *comp)
    {
-   return false;
+   uint32_t flags8 = self()->getX86ProcessorFeatureFlags8(comp);
```

Comment:
Also does this Flags8 need to be guarded for older hardware that may not have this many flags? ie when was flags8 introduced and will we hit any problems in old hardware if we attempt to read it?